### PR TITLE
improve y labels in plots

### PR DESCRIPTION
**Pull request by ggrrll**
here is my proposition to improve the plot:

- rotate and move a little -- to avoid overlap
- add a `_` to IMF names to make it more clear

#52

### CODE REVIEW

File: pyhht/visualization.py [hunk 1]
```diff
@@ -72,7 +73,8 @@ def plot_imfs(signal, imfs, time_samples=None, fignum=None, show=True):
     ax.tick_params(which='both', left=False, bottom=False, labelleft=False,
                    labelbottom=False)
     ax.grid(False)
-    ax.set_ylabel('Signal')
+    ax.set_ylabel('Signal',rotation=0)
```
Comment (jaidevd) [inline pyhht/visualization.py]: Please expose the `rotation` parameter in the function definition so others can use it as they like. Keep the default as 0. Also please run `flake8` on the file and fix all related issues.

File: pyhht/visualization.py [hunk 2]
```diff
@@ -72,7 +73,8 @@ def plot_imfs(signal, imfs, time_samples=None, fignum=None, show=True):
...
+    ax.yaxis.set_label_coords(-0.1,0)
```
Comment (jaidevd) [inline pyhht/visualization.py]: These parameters should also be exposed in the `plot_imfs` function, with these values as default.

**Review by jaidevd — CHANGES_REQUESTED** [review]

**Comment by jaidevd** [comment]
Please also make sure that Travis CI is passing.

**Comment by ggrrll** [comment]
so, I add the flake8 test and the exposures  -- didn't have time for Travis CI (sorry, I am also quite new to these tests :) )

**Comment by jaidevd** [comment]
Thanks a lot for this, @ggrrll 

It seems that the tests failing in travis are the plotting tests. Makes sense, since we have changed the plotting code here, but not the template images against which the generated plots are compared. Please take a look at this whenever you have time: https://matplotlib.org/1.5.3/devel/testing.html#writing-an-image-comparison-test

And let me know. Then we'll discuss how to fix these tests for the current changes.

Thanks again,

**Comment by ggrrll** [comment]
sorry, my fault - 
[e76a08e](https://github.com/jaidevd/pyhht/pull/53/commits/e76a08e045824ffe779f2cc1108d17884ec57d26) is related to #54

**Comment by jaidevd** [comment]
Please roll back the last commit, I've already fixed that reference in https://github.com/jaidevd/pyhht/commit/f4e6a044e57d34fc95f5bd2651ef2e232ce2d42b

**Comment by jaidevd** [comment]
Closing for lack of activity.

**Pull request closed without merging**